examples/demo-env/atmos.yaml

```
@@ -0,0 +1,21 @@
+base_path: "./"
+
+components:
+  terraform:
+    base_path: "components/terraform"
```

After you move this to `tests/scenarios/env`, then change this to
```suggestion
    base_path: "../../components/terraform"
```